parsers/mirex05melody_parser.py

```diff
@@ -45,33 +50,33 @@ def fill_annotation_metadata(annot):
     annot.annotation_metadata.annotation_rules = ""
     annot.annotation_metadata.validation = "TODO"
```

Comment:
Well, I don't think anyone ever validated these annotations (neither adc2004 or mirex05) outside of the person who created them, so depending on how you interpret this field it could probably stay TODO forever. Silliness aside, I've just changed it to set empty string (for both datasets), which is probably less confusing: fb41e3a